src/bot/onMessage.ts

```diff
@@ -15,7 +15,7 @@ export default async function onMessage(message: IMessage): Promise<void> {
   const uniqueWords = message.cleanContent.toLocaleLowerCase().split(' ').length
   const lastWords = message.cleanContent.toLocaleLowerCase().split(' ').slice(uniqueWords - 5, uniqueWords )
 
-  const offendingWord = flaggedWords.find(f => firstWords.some(w => w.includes(f))) || flaggedWords.find(f => lastWords.some(w => w.includes(f)))
+  const offendingWord = flaggedWords.find(f => firstWords.includes(f) || flaggedWords.find(f => lastWords.includes(f)))
```

Comment:
Speaking of "offending," is there a closing parenthesis issue here? Line breaks below added for readability to find the offensive parenthesis.

```js
const offendingWord = 
  flaggedWords.find(f => firstWords.includes(f)) || 
  flaggedWords.find(f => lastWords.includes(f))
```

Reply:
Ahhh, good catch.

Reply:
![bitmoji](https://sdk.bitmoji.com/render/panel/9ff55377-6e8f-4a55-9baf-471d1ee75e6a-b24761f5-99bf-4885-9465-63838b10c2c3-v1.png?transparent=1&palette=1&width=246)